### ISSUE TITLE
Migrate Jenkins jobs from /cu to /glassfish

### ISSUE THREAD

**Issue by pzygielo** [eclipse-ee4j/glassfish-concurro#63]
I understand that https://ci.eclipse.org/cu/ instance is for specification project, and after split implementation jobs should be recreated in https://ci.eclipse.org/glassfish/ instance.

- [ ] [concurro-build-master](https://ci.eclipse.org/cu/job/concurro-build-master/)
- [x] [concurro_1-build-and-stage](https://ci.eclipse.org/cu/job/concurro_1-build-and-stage/)
- [ ] [concurro_3-staging-to-release](https://ci.eclipse.org/cu/job/concurro_3-staging-to-release/)

**Comment by arjantijms** [comment]
Agree. I will create a job for this on the https://ci.eclipse.org/glassfish/ (or if you'd rather do it, go ahead of course).

**Comment by pzygielo** [comment]
I see https://ci.eclipse.org/glassfish/job/1_concurro-build-and-stage/

Not sure about the remaining two - are they not needed and can this issue be closed?

**Comment by pzygielo** [comment]
I suppose current workflow will do for [concurro-build-master](https://ci.eclipse.org/cu/job/concurro-build-master/) and releasing can be handled by generic job - hence closing.
